test/query.js

```diff
@@ -790,33 +796,36 @@ describe('Query', () => {
         }
       `
       const fixture = {
-        email: 'demo@demo.com',
-        user: {
-          email: 'newemail@demo.com'
-        },
-        familyInfo: [{
-          isLocal: true,
-          father: {
-            email: 'father@demo.com'
-          }
-        },
-        {
-          id: '1',
-          isLocal: false,
-          father: {
-            id: '100'
-          }
-        }]
+        getMe: {
+          email: 'demo@demo.com',
+          user: {
+            email: 'newemail@demo.com'
+          },
+          familyInfo: [{
+            isLocal: true,
+            father: {
+              email: 'father@demo.com'
+            }
+          },
+          {
+            id: '1',
+            isLocal: false,
+            father: {
+              id: '100'
+            }
+          }]
+        }
       }
 
       const { getMe } = tester.mock({
         query: query,
         fixture
       })
+      console.log('-->', getMe)
```

Comment:
maybe remove this?

Reply:
Ohh!! Thank you so much!! that should be removed